src/tests/multihost/alltests/conftest.py

```diff
@@ -963,13 +963,6 @@ def create_posix_usersgroups_failover(session_multihost):
                                   group_info)
         except LdapException:
             assert False
-            group_dn = 'cn=ldapusers,ou=Groups,dc=example,dc=test'
```

Comment:
I would try to figure out what was the intention.

Reply:
You were right. I spoke with Steeve and the code was wrongly indented.